Source: maxice8
Upstream: not upstreamable
Reason: glib was chagned to use meson which doesn't provide aclocal this breaks gtk+3 and other packages that uses aclocal.

--- configure.ac
+++ configure.ac
@@ -798,12 +798,6 @@ esac
 # Run AM_PATH_GLIB_2_0 to make sure that GLib is installed and working
 #
 
-AM_PATH_GLIB_2_0(glib_required_version, :,
-  AC_MSG_ERROR([
-*** GLIB glib_required_version or better is required. The latest version of
-*** GLIB is always available from ftp://ftp.gtk.org/pub/gtk/.]),
-  gobject gmodule-no-export)
-
 PKG_PROG_PKG_CONFIG_FOR_BUILD
 GLIB_CFLAGS_FOR_BUILD=`$PKG_CONFIG_FOR_BUILD --cflags glib-2.0 gobject-2.0 gmodule-no-export-2.0`
 GLIB_LIBS_FOR_BUILD=`$PKG_CONFIG_FOR_BUILD --libs glib-2.0 gobject-2.0 gmodule-no-export-2.0`
